diagnose/diagnose.go

```diff
@@ -81,7 +82,7 @@ func (n *Server) EnableDebug(ip string, port int) {
 	// go func() {
 	// 	http.Serve(n.sk, n.mux)
 	// }()
-	http.ListenAndServe(":8000", n.mux)
```

Comment:
nit: you could also do `http.ListenAndServe(fmt.Sprintf(":%d", port), n.mux)`

Reply:
done